### DISCUSSION TITLE
Move to different team resource operation

### DISCUSSION THREAD

**Discussion by gavry** [coollabsio/coolify#2029] [Feature Requests]
We've set up a bunch of resources under the root team (foolishly), but now would like to allow granular access to other members. It seems our only option is to recreate the resources under the other teams, but we'd love the ability to move the resource to a project in another team. Maybe by showing all projects of all teams that the current user is an admin of? An additional (but prob unnecessary) convenience could be to move a whole project to a different team.

Edit:
This is possibly a larger lift than expected because servers aren't currently shareable between teams:
https://github.com/coollabsio/coolify/discussions/1820

Which led me to here:
https://github.com/coollabsio/coolify/blob/main/app/Models/Server.php

(if I understand correctly, Server has a many-to-one relationship with Team? Which means this may require the creation of a connector table to allow for many-to-many)

**Comment by adiologydev** [comment]
Same here, really need this feature for more granular control and access management.

**Comment by biboc** [comment]
Same here

**Comment by kolezka** [comment]
Bump!!

**Comment by reachcreator** [comment]
Ditto just run into this issue.

**Comment by hnykda** [comment]
Yeah, would be great to have it

**Comment by prestonzen** [comment]
Same here. Would like to pass projects and services across teams. R&D team and prod team for example.

**Comment by sjmog** [comment]
This would be super-helpful since projects seem like more of an organisation tool than necessarily needing to represent the underlying application structure.

**Comment by jcarasick-geog8r** [comment]
same here

**Comment by rao-manoj** [comment]
badly needed

**Comment by alandolsi** [comment]
same here

**Comment by hungchihsueh** [comment]
up up

**Comment by jjuannn** [comment]
+1

**Comment by stevebrainng** [comment]
I will spend some time to work and summit a pull request for this.

**Comment by adamkdean** [comment]
We really do miss this

**Comment by hammashamzah** [comment]
+1

**Comment by lewismunene020** [comment]
I also  really need this.
Who is working on a solution we collaborate and create a  pr ?  @stevebrainng

**Comment by adiologydev** [comment]
This feature will be shipped with v5, which is in progress.

**Reply by lewismunene020** [reply]
Thanks @adiologydev  for the response.

Will be waiting for it

**Reply by xorweak** [reply]
Amazing! thanks. I'm waiting for it too.

**Reply by stevebrainng** [reply]
> This feature will be shipped with v5, which is in progress.

Hello, @adiologydev 

Thanks for the work, I didn't have some extra time to spare for the solution, 

How's coming by?

**Comment by sabbajohn** [comment]
Anyone tried to transfer (servers, apps etc) directly on data base ? Did it worked?

**Reply by zawoj** [reply]
I did -and it worked. There will be a post about it soon.

**Comment by mcanti** [comment]
bump

**Comment by tquiviger** [comment]
Really interested by this feature, any news?

**Comment by parisosuch** [comment]
Commenting to bump this. Would be nice to transfer projects (and resources) to other teams. An easier win would be transferring if on the same local server.

**Comment by estevan-ulian** [comment]
UP

**Comment by grrinch** [comment]
+1 great idea, just needed to do this!

**Comment by facundopadilla** [comment]
+1

**Comment by Shadow1363** [comment]
+1, also foolishly set everything up on Root Team

**Comment by mathzdev** [comment]
+1 haha
